### ISSUE TITLE
Normalisation of the ground-truth smallest eigenvector

### ISSUE THREAD

**Issue by ivan-pavlov** [vcg-uvic/learned-correspondence-release#15]
Hello!

Thank you for the code release for "Eigendecomposition-free Training of Deep Networks with Zero Eigenvalue-based Losses" paper.

In the paper, you desribe a normalisation of the ground-truth smallest eigenvector:
<img width="719" alt="image" src="https://user-images.githubusercontent.com/11914755/60627275-cfc33e80-9ded-11e9-95e2-898074c4d191.png">

I can see how you are building the ground-truth smallest eigenvector for the loss computation [here](https://github.com/Dangzheng/Eig-Free-release/blob/43360d581bd1c1346ff06689cc40deaf6ad2d80b/network.py#L150). 

But, unfortunately, I could not find in the loss function or elsewhere in the code how you scale and translate this eigenvector.

Could you please clarify to me how you perform the normalisation of the ground-truth smallest eigenvector?

**Comment by kmyid** [comment]
@Dangzheng Can you reply? I guess we are quite late on replying though.
